devicetypes/smartthings/inovelli-dimmer.src/inovelli-dimmer.groovy

```diff
@@ -0,0 +1,599 @@
+/*	Copyright 2020 SmartThings
+*
+*	Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except
+*	in compliance with the License. You may obtain a copy of the License at:
+*
+*		http://www.apache.org/licenses/LICENSE-2.0
+*
+*	Unless required by applicable law or agreed to in writing, software distributed under the License is distributed
+*	on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License
+*	for the specific language governing permissions and limitations under the License.
+*
+*	Inovelli Dimmer
+*
+*	Copyright 2020 SmartThings
+*
+*/
+metadata {
+	definition (name: "Inovelli Dimmer", namespace: "smartthings", author: "SmartThings", ocfDeviceType: "oic.d.switch", runLocally: false, minHubCoreVersion: '000.017.0012', executeCommandsLocally: false, genericHandler: "Z-Wave", mcdSync: true) {
```

Comment:
`runLocally `and `executeCommandsLocally `are _false_ by default, there's no actual need to include them here. Also I'd remove `genericHandler`.

Reply:
Done